### PR TITLE
Add Leaflet.HighlightableLayers and Leaflet.DraggableLines plugins to docs

### PR DESCRIPTION
**Pull request by cdauth**

### CODE REVIEW

**Comment by johnd0e** [comment]
@cdauth 
Please resolve conflicts

File: docs/plugins.md [hunk 1]
```diff
@@ -1867,6 +1867,17 @@ These plugins provide new markers or news ways of converting abstract data into
 			<a href="https://github.com/coyotesqrl/">R.A. Porter</a>
 		</td>
 	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/FacilMap/Leaflet.HighlightableLayers">Leaflet.HighlightableLayers</a>
+		</td>
+		<td>
+		    Highlight Leaflet lines and polygons by adding a border and raising them above others. Add a transparent border to increase the tolerance for mouse/touch interactions. <a href="https://unpkg.com/leaflet-highlightable-layers/example.html">Demo</a>
```
Comment (Falke-Design) [inline docs/plugins.md:1875]: Please replace blank with tab
```suggestion
			Highlight Leaflet lines and polygons by adding a border and raising them above others. Add a transparent border to increase the tolerance for mouse/touch interactions. <a href="https://unpkg.com/leaflet-highlightable-layers/example.html">Demo</a>
```

File: docs/plugins.md [hunk 2]
```diff
@@ -2542,6 +2553,17 @@ Allows users to create, draw, edit and/or delete points, lines and polygons.
 			<a href="https://github.com/Lemaf">Lemaf</a>
 		</td>
 	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/FacilMap/Leaflet.DraggableLines">Leaflet.DraggableLines</a>
+		</td>
+		<td>
+		    Add/move/remove points on routes, lines and polygons by drag&drop. <a href="https://unpkg.com/leaflet-draggable-lines/example.html">Demo</a>
```
Comment (Falke-Design) [inline docs/plugins.md:2561]: Please replace blank with tab
```suggestion
			Add/move/remove points on routes, lines and polygons by drag&drop. <a href="https://unpkg.com/leaflet-draggable-lines/example.html">Demo</a>
```

**Review by Falke-Design — CHANGES_REQUESTED** [review]

**Comment by cdauth** [comment]
@johnd0e @Falke-Design Fixed.

**Review by Falke-Design — APPROVED** [review]
- [x] Tabs used
- [x] All Links are working

@jonkoops kan be merged

**Review by johnd0e — APPROVED** [review]

**Pull request merged**